pkg/bootstrap/cni.go

```diff
@@ -37,19 +46,27 @@ func internalInstallCNI(ctx context.Context, cl client.Client, config cni.Config
 	values["imagePullSecrets"] = config.ImagePullSecrets
 	// TODO: imagePullPolicy, resources
 
-	// always install the latest version of the CNI image
-	renderings, _, err := helm.RenderChart(path.Join(versions.DefaultVersion.GetChartsDir(), "istio_cni"), operatorNamespace, values)
-	if err != nil {
-		return err
+	var releases []string
+	for _, version := range supportedVersions {
+		releases = append(releases, version.String())
 	}
+	values["supportedReleases"] = releases
+
+	// always install the latest version of the CNI image
+	renderings, _, err = helm.RenderChart(path.Join(versions.DefaultVersion.GetChartsDir(), "istio_cni"), operatorNamespace, values)
+	return
```

Comment:
Ideally, this function should return `renderings["istio_cni"]` instead of the entire `renderings` map.

Reply:
That behaviour was already present in the code, it has not changed with this PR

Reply:
Sorry, when I expanded this I see you are talking about the new function

Reply:
I can see what you mean, keep the knowledge about the key ("istio_cni") in one function instead of reference it in two.  I can submit a change for this later and apply it to both 2.0 and 2.1 branches.

Reply:
Yes, that's what I meant.